test/__snapshots__/codegen.test.ts.snap

```diff
@@ -1,4 +1,4 @@
-// Vitest Snapshot v1
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
```

Comment:
After updating vitest, this changed, since when running tests is autogenerated uploading here